src/i18n/languages/ro.json

```diff
@@ -28,6 +28,18 @@
   "language-english": "Engleza",
   "language-romanian": "Romana",
   "edit": "Editat",
+  "signUp.title": "Creeaza un cont",
+  "signUp.firstName": "Completati cu prenumele dvs",
+  "signUp.lastName": "Completati cu  numele dvs",
+  "signUp.email.valid": "Adaugati un email valid",
+  "signUp.email.required": "Emailul este necesar",
+  "signUp.password.valid": "Parola trebuie sa fie mai lunga de 8 caractere",
+  "signUp.password.required": "Parola este necesara",
+  "button.create": "Creeaza",
+  "first.name": "Prenume",
```

Comment:
I suggest keeping the translation keys more consistent and structured. Once more features are added, this file will get a lot bigger and it'll be hard to manage all the keys and where they have been used.

For example, this is used in the signup page but you didn't keep the same structure of `signUp.firstName` as the keys above. You could do something like `signUp.firstName.label` then `signUp.firstName.required` for the validation required error message.